Tasks/UseGoV1/task.json

```diff
@@ -0,0 +1,66 @@
+{
+    "id": "334727F4-9495-4F9D-A391-FC621D671474",
+    "name": "UseGo",
+    "friendlyName": "Use Go",
+    "description": "Sets up a Go environment and installs it to the PATH. Additionally provides proxy support (except for go get which is done through source control providers).",
+    "helpUrl": "https://go.microsoft.com/fwlink/?linkid=867581",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=867581)",
+    "category": "Tool",
+    "preview": true,
+    "ecosystem": "go",
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "satisfies": [
+        "GO"
+    ],
+    "demands": [],
+    "instanceNameFormat": "Use Go $(version)",
+    "groups": [
+        {
+            "name": "advanced",
+            "displayName": "Advanced",
+            "isExpanded": false
+        }
+    ],
+    "inputs": [
+        {
+            "name": "version",
+            "type": "string",
+            "label": "Version",
+            "required": false,
+            "helpMarkDown": "The Go version to download (if necessary) and use. Example: 1.9.3"
```

Comment:
Does the user need to know this includes downloading? Description reads weird to me. @vtbassmatt to comment.

Reply:
I think that's good to know since some use tasks don't download if its not there (and thus behave differently)